Gordon360/Services/HousingService.cs

```diff
@@ -0,0 +1,774 @@
+using Gordon360.Exceptions.CustomExceptions;
+using Gordon360.Models;
+using Gordon360.Models.ViewModels;
+using Gordon360.Repositories;
+using Gordon360.Services.ComplexQueries;
+using Gordon360.Static.Data;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+
+namespace Gordon360.Services
+{
+    public class HousingService : IHousingService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+
+
+        public HousingService(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+        /// <summary>
+        /// Calls a stored procedure that returns a row in the staff whitelist which has the given user id,
+        /// if it is in the whitelist
+        /// </summary>
+        /// <param name="userID"> The id of the person using the page </param>
+        /// <returns> Whether or not the user is on the staff whitelist </returns>
+        public bool CheckIfHousingAdmin(string userID)
+        {
+            IEnumerable<HousingAdminViewModel> idResult = null;
+
+            SqlParameter idParam = new SqlParameter("@USER_ID", userID);
+
+            idResult = RawSqlQuery<HousingAdminViewModel>.query("GET_AA_ADMIN @USER_ID", idParam);
```

Comment:
Is there a reason to declare the `RawSqlQuery` result separately from assigning it? It seems more long-winded and confusing than is necessary.
```suggestion
            SqlParameter idParam = new SqlParameter("@USER_ID", userID);

            var idResult = RawSqlQuery<HousingAdminViewModel>.query("GET_AA_ADMIN @USER_ID", idParam);
```

Reply:
You seem to do this habitually. While it isn't wrong, it does bloat the code quite substantially, and in my opinion it hinders readability more than helping it.